lib/data/multipleBackendGateway.js

```diff
@@ -17,6 +17,45 @@ function createLogger(reqUids) {
 }
 
 const clients = parseLC(config);
+let multBackendResp = {};
```

Comment:
It's being used in both the checkAwsHealth function and the gateway healthcheck function